sharktank/sharktank/torch_exports/sdxl/unet.py

```diff
@@ -280,16 +281,6 @@ def get_punet_model(hf_model_name, external_weight_path, quant_paths, precision=
         revision = "efda8afb35fd72c1769e02370b320b1011622958"
 
     # TODO (monorimet): work through issues with pure fp16 punet export. Currently int8 with fp8/fp8_ocp/fp16 sdpa are supported.
-    # elif precision != "fp16":
```

Comment:
Is this TODO still useful now that the code below it is deleted?

Reply:
Yeah, because it's still a missing piece in functionality.